src/Sylius/Bundle/ResourceBundle/DependencyInjection/Configuration.php

```diff
@@ -174,4 +175,35 @@ private function addTranslationsSection(ArrayNodeDefinition $node)
             ->end()
         ;
     }
+
+    private function addDriversSection(ArrayNodeDefinition $node)
```

Comment:
This method is copied verbatim from the GridBundle. Not really sure how to avoid that without adding a new shared dependency just for this section of code..